qiskit/transpiler/passes/optimization/optimize_1q_decomposition.py

```diff
@@ -123,20 +198,19 @@ def _substitution_checks(self, dag, old_run, new_circ, new_basis):
             )
```

Comment:
Should we adjust this message and include the fidelities for ease of debugging user reports?